### PR TITLE
Added cluster check entry about Java7

### PR DESCRIPTION
**Pull request by celaus**

### CODE REVIEW

File: blackbox/docs/sql/system.txt [hunk 1]
```diff
@@ -937,6 +937,8 @@ There are currently four checks:
 +------------------+------------------------------------+
 | 5                | Number of partitions.              |
 +------------------+------------------------------------+
+| 6                | Java 7 deprecation warning.        |
```
Comment (seut) [inline blackbox/docs/sql/system.txt]: pls change the description above this table, it currently writes about 4 checks ;) (maybe just reword it to skip number of checks so we won't have to change it always)

**Comment by seut** [comment]
ack

File: blackbox/docs/sql/system.txt [hunk 2]
```diff
@@ -922,7 +922,7 @@ The table schema is the following:
 |                  | check for the setting has passed.|             |
 +------------------+----------------------------------+-------------+
 
-There are currently four checks:
+Currently Crate checks the following:
```
Comment (dobe) [inline blackbox/docs/sql/system.txt]: @celaus pls use a select statement to generate this table
